docs/flipflop/t_flipflop.md

```diff
@@ -47,9 +47,8 @@ From the above characteristic table, the next state equation can be directly wri
 
 
 ```yaml
-          Q(t+1)=T′Q(t)+TQ(t)′
-           
-          ⇒  Q(t+1)=T⊕Q(t)
+          Q(t +1 ) = T'.Q(t) + T.Q(t)'
```

Comment:
Q(t + 1) = T'.Q(t) + T.Q(t)'
Remove space after 1 and add a space before it as I have done

Reply:
Sure, thanks :+1: